docs/tendermint-rpc.md

```diff
@@ -0,0 +1,210 @@
+# Tendermint RPC
```

Comment:
Can you please add a reference to this doc to https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/README.md#tendermint-rpc-and-light-client and https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/docs/transactions.md#how-to-read-from-the-ledger (Tendermint RPC item)